Sources/MessagingInApp/Gist/Managers/MessageManager.swift

```diff
@@ -71,6 +71,11 @@ class MessageManager {
         // Expect subclass implements this.
     }
 
+    // Called when custom action button is tapped.
+    func onInlineButtonAction(message: Message, currentRoute: String, action: String, name: String) {
+        // Expect subclass implements this.
+    }
```

Comment:
Do we need to add any code to the `MessageManager`? [The inline message manager already gets a callback when the action button is clicked](https://github.com/customerio/customerio-ios/pull/763/files#diff-f93ca71bad51e58601b7bc5e6ae8db007cbdbe9ca700730d611472245b322e4bR64) so I don't think we need to add a function to this class? 

Also, this class is designed to be generic without knowledge of modal or inline messages. From the name of the function, this function looks to me like it should belong only in the inline message manager.

Reply:
Ah ! We don't need it. Thank you.